### ISSUE TITLE
Manage access does not let a resource be shared as edit or view with an admin user when there is only one owner

### ISSUE THREAD

**Issue by dtarb** [hydroshare/hydroshare#4138]
To reproduce this create a resource.
Open manage access.
select a user with admin permissions (e.g. pkdash or dtadmin)
Leave access level as can view
Click add
The following error occurs
![image](https://user-images.githubusercontent.com/3456833/108611081-38cb4180-7398-11eb-981b-4ae4b50b197b.png)

**Comment by pkdash** [comment]
@dtarb Not sure what should be the expected behavior. An admin always has owner role on any resource though it is not displayed in the manage access dialog box. I currently have a fix for this issue where this error is not displayed and the admin can be assigned  any of the roles (view, edit, owner). Once the admin is assigned one of these roles, admin is listed in the manage access panel as one of the users having access to the resource. Let me know if that's the desired behavior.

**Comment by dtarb** [comment]
@pkdash yes this seems to be good behavior to have as the desired behavior.  I think we should end up with the following
- Admin user always is able to act on a resource as if they are owner.
- If a user assigns a role to an admin user, then this role shows up in the manage access dialog listing
- If a user removes a role from an admin user, then the role is removed from the listing in manage access, but the admin user is still able to act on the resource by virtue of being an admin.
- Resources where an admin user has been assigned a role, should show up in the shared with me part of the admin users My Resources page.

**Comment by pkdash** [comment]
@dtarb Sounds good.
